core/trino-main/src/main/java/io/trino/security/AccessControlManager.java

```diff
@@ -205,6 +211,26 @@ public void loadSystemAccessControl(String name, Map<String, String> properties)
         setSystemAccessControls(ImmutableList.of(systemAccessControl));
     }
 
+    private SystemAccessControlContext createContext(String systemAccessControlName)
+    {
+        return new SystemAccessControlContext()
+        {
+            private final Tracer tracer = openTelemetry.getTracer("trino.system-access-control." + systemAccessControlName);
```

Comment:
nit: verify that systemAccessControlName is not empty or null

Reply:
this is verified by the caller within the class and the method is private 
also, empty string is currently valid factory name :)